### DISCUSSION TITLE
Transfer domain

### DISCUSSION THREAD

**Discussion by astnai** [vercel/community#6845] [Help]
Problem solved!

**Comment by versecafe** [comment]
@astnai When you go to `https://vercel.com/[team]/~/domains` does the domain not appear at all?

**Reply by astnai** [reply]
> @astnai When you go to `https://vercel.com/[team]/~/domains` does the domain not appear at all?

Yes, it appears. The domains is in "Thrid Party". It's supposedly being transferred
